src/content/test/text-legibility/guidance.tsx

```diff
@@ -40,17 +57,57 @@ export const guidance = create(({ Markup, Link }) => (
                         ratio (for example, white text on a sky-blue background).
                     </li>
                 </ul>
+
+                <h3>
+                    Make sure users can increase the spacing between letters, words, lines of text, and paragraphs with no clipping or
+                    overlapping. (<Link.WCAG_1_4_12 />)
+                </h3>
+                <ul>
+                    <li> All text must respond to user-initiated changes in spacing.</li>
+                    <li> All text must remain visible, with no clipping or overlapping.</li>
+                </ul>
+
+                <h3>
+                    Make sure users can zoom the browser to 400% and still read the text without having to scroll in two dimensions. (
+                    <Link.WCAG_1_4_10 />)
+                </h3>
+                <ul>
+                    <li> Ideally, scripts read horizontally should require only vertical scrolling, and </li>
+                    <li> Scripts read vertically should require only horizontal scrolling.</li>
+                </ul>
+
+                <h3>
+                    Make sure content that appears on focus or hover is dismissible, hoverable, and persistent. (<Link.WCAG_1_4_13 />)
```

Comment:
This seems a bit out of place here; was there intended to be something else here?

Reply:
nope, this requirement is part of this test